app/views/talks/_talk.html.erb

```diff
@@ -69,3 +69,32 @@
     <span>Edit</span>
   <% end %>
 </div>
+
+<div class="py-4"></div>
+
+<%= link_to event_path(talk.event), class: "card bg-gray-200 w-full xl:flex xl:flex-row xl:px-6 xl:py-6 xl:gap-4 overflow-hidden" do %>
+  <div class="flex aspect-video xl:w-1/5 overflow-hidden">
+    <% event_talk = talk.event.talks.where.not(id: talk.id).sample || talk.event.talks.take %>
+    <%= image_tag event_talk.thumbnail_sm,
+          srcset: ["#{event_talk.thumbnail_lg} 2x"],
+          id: dom_id(event_talk),
+          height: 174,
+          width: 310,
+          alt: "explore all talks recorded at #{talk.event.name}",
+          style: "view-transition-name: #{dom_id(event_talk, :image)}",
```

Comment:
this should be remove otherwise it breaks view-transition. view transition name must be uniq on the page